subprojects/emcee/queue-api/src/main/kotlin/com/avito/emcee/queue/Device.kt

```diff
@@ -0,0 +1,15 @@
+package com.avito.emcee.queue
+
+import com.squareup.moshi.Json
+
+public data class Device(
+    /**
+     * Should be deleted
+     */
+    @Json(name = "deviceType")
+    val type: String,
+    /**
+     * Representing a emulator api version i.e. 21,22,29,30 etc.
+     */
```

Comment:
Let's use `@Json` for all json fields.
It's too easy to break a contract while refactoring.

Reply:
I think it's an overhead. I think we will rarely change those models. And they are placed in a separate module

Reply:
The motivation is don't even think about it while changing any code